docs/mdbook/src/android/getting-started-guide.md

```diff
@@ -0,0 +1,118 @@
+# Quickstart
+
+1. Add bintray Maven repositories to your project-level Gradle file (usually `<project>/<app-module>/build.gradle`).
+
+    ```gradle
+        allprojects {
+            repositories {
+                ...
+                mavenCentral()                
+            }
+        }
+    ```
+
+2. Add the library as a dependency into your module Gradle file (usually `<project>/<app-module>/build.gradle`). Please replace `<version>` with a valid MapLibre GL version (e.g.: `org.maplibre.gl:android-sdk:10.0.2`).
```

Comment:
```suggestion
2. Add the library as a dependency into your module Gradle file (usually `<project>/<app-module>/build.gradle`). Replace `<version>` with the latest MapLibre GL Native version (e.g.: `org.maplibre.gl:android-sdk:10.0.2`).
```

Reply:
Could you add a link to where you can find what the latest version is?

Reply:
I added the link that refers to the maven repository of MapLibre GL Native for Android so that  we can find the latest version in this commit 6385569ac80a64362250d25cd55cc29bbc604465.